### DISCUSSION TITLE
BottomNavigation custom height

### DISCUSSION THREAD

**Discussion by spexa-io** [callstack/react-native-paper#2428] [Q&A] [answered]
We need to be able to change the height of BottomNavigation. Because we had a problem when we used the custom label. Is there a solution to this issue?

![Screen Shot 2020-12-14 at 10 01 18 PM](https://user-images.githubusercontent.com/75990401/102120522-e7fb1200-3e57-11eb-89e4-2a55ab209b99.png)

**Comment by raajnadar** [comment, marked as answer]
Try adding this prop

```
barStyle={{
   height: 80,
   backgroundColor: '#694fad',
   justifyContent: 'center',
}}
```
